source/docs/03-metadata.xml

```diff
@@ -1392,9 +1392,19 @@
                         <specDesc key="colophon"/>
                      </specList>
                      </p>
-                  <p>The <gi scheme="MEI">condition</gi> element may contain text, or child elements that may be used for structuring the description (e.g., for linking) to accurately describe the physical state or condition of a source.</p>
-                     <p>The condition can thus be described in general (<gi scheme="MEI">physDesc</gi>) or in specific contexts (see “contained by” section of the element definition).</p>
-                     <p>For describing the condition of parts of a source (e.g., the binding) the <gi scheme="MEI">condition</gi> element can be used on the specific level. The level of detail you provide will depend on your specific needs for accurately recording the material condition.</p>
+                  <p>Encoding general information about the condition of a source.</p>
+                   <p>
+                      <specList>
+                        <specDesc key="condition"/>
+                      </specList>
+                   </p>
+                     <p>The element <gi scheme="MEI">condition</gi> may contain either plain text or elements that can be used to structure the description (e.g. for linking) in order to describe the state or condition of a source.</p>
+                     <p>The highest hierarchical level to describe the condition, in general, is at <gi scheme="MEI">physDesc</gi>. For a detailed description of special interest, the element <gi scheme="MEI">condition</gi> can also be used on lower hierarchical levels (see section “contained by” in the element definition).</p>
+                     <p>The <gi scheme="MEI">condition</gi> element as a direct child of <gi scheme="MEI">physDesc</gi> should be used to describe the condition of a source. To describe special concerns, such as the condition of parts or aspects of a source (e.g. the binding) the <gi scheme="MEI">condition</gi> element can also be used for a more detailed description of that part or aspect (e.g. at <gi scheme="MEI">binding</gi> ). The detail you provide, and how the descriptions are structured, depend on your own encoding guidelines.</p>
```

Comment:
```suggestion
                     <p>The <gi scheme="MEI">condition</gi> element as a direct child of <gi scheme="MEI">physDesc</gi> should be used to describe the condition of a source. To describe special concerns, such as the condition of parts or aspects of a source (<abbr>e.g.</abbr>, the binding) the <gi scheme="MEI">condition</gi> element can also be used for a more detailed description of that part or aspect (<abbr>e.g.</abbr>, at <gi scheme="MEI">binding</gi> ). The detail you provide, and how the descriptions are structured, depend on your own encoding guidelines.</p>
```

Reply:
(I added my suggestions to @musicEnfanthen's)